inc/TRestAxionOptics.h

```diff
@@ -41,20 +41,44 @@ class TRestAxionOptics : public TRestMetadata {
     /// A vector containing the shells ring radius definitions. First element is the lower radius.
     std::vector<std::pair<Double_t, Double_t>> fShellsRadii;  //<
 
+    /// The angle between two consecutive spider arms measured in radians.
+    Double_t fSpiderArmsSeparationAngle = 0;  //<
+
+    /// The position angle at which the spider arm will start
+    Double_t fSpiderOffsetAngle = 0;  //<
+
+    /// The width of each specific spider arm. Measured in radians. Default is 2.5 degrees.
+    Double_t fSpiderWidth = TMath::Pi() / 18. / 4.;  //<
+
+    /// The spider structure will be effective from this radius, in mm. Default is from 0 mm.
+    Double_t fSpiderStartRadius = 0.;  //<
```

Comment:
Maybe we should allow for a hole here as there are no mirrors in the middle anyways and we are planning it for the optics alignment. So maybe the default should be greater than 0mm (maybe like 20mm as the innermost radius of the new BabyIAXO optics probably can't be smaller than 54mm).

Reply:
Sure, thats the idea for that data member. Yes, probably is a good idea to include a non-zero default value.

I introduced it here:
906028d44bbd82965ba7d4cd1b2cbd7b784c2da4

Change it at will.